index.html

```diff
@@ -38,16 +38,17 @@
 			<div id="qcSlideText">
 				<h1 id="slidecaption">2017 conference</h1>
 				<h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
-
-				<div id="slidedesc">
-					<p>
-						Women technologists, let's take your career to the next level!
-					<p>
-					<p>
-					 Join us for 4 days designed to polish your skills and fully own your expertise through writing, speaking, and open source.
-				</p>
+				<div>
+					<p>Women technologists, let's take your career to the next level!<br class="hidden-on-xs"> Join us for 4 days designed to polish your skills and fully own your expertise through writing, speaking, and open source.</p>
+				</div>
+				<div class="row cfp-banner">
+					<div class="col-xs-12 col-sm-6 cfp">
+						Call for Proposals now open. Apply by 4/23.
+					</div>
+					<div class="col-xs-12 col-sm-6 submit-talk">
+						<a href="https://www.papercall.io/writespeakcode" target="_blank">Submit Your Talk</a>
+					</div>
 				</div>
-
```

Comment:
nitpick: can we add this space back to create more obvious separation from the timer section?